### PR TITLE
readme.md has been updated

### PR DESCRIPTION
**Pull request by sf-jeff-lim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,333 @@
...
+### Try the sample app
+
+The fastest way to test the Chat SDK is to build your chat app on top of our sample app. To create a project for the sample app, download the app from our GitHub repository. The link is down below. 
+
+- https://github.com/sendbird/Sendbird-JavaScript
+
+You can also download the sample using a git command:
+
+```bash
+$ git clone https://github.com/sendbird/SendBird-JavaScript
```
Comment (chrisallo) [inline README.md:71]: It'd be good to add the live sample page https://sample.sendbird.com imho.

File: README.md [hunk 2]
```diff
@@ -1,68 +1,333 @@
...
+```bash
 import * as SendBird from "sendbird";
 var sb = new SendBird({ appId: "APP_ID" });
 // do something...
 ```
 
-If you have trouble importing `Sendbird`, please check your `tsconfig.json` file and change the value of `"allowSyntheticDefaultImports"` to `true` in `compilerOptions`.
+If you have trouble importing Sendbird, please check your `tsconfig.json` file and change the value of `allowSyntheticDefaultImports` to true in `compilerOptions`.
+
+- **Bower**
```
Comment (chrisallo) [inline README.md:153]: `bower` is quite old-fashioned way to install a package. How about adding `yarn add sendbird` instead?

**Review by chrisallo — CHANGES_REQUESTED** [review]

**Comment by chrisallo** [comment]
And the samples would be renewed soon. I shall let you know when they are ready.

File: README.md (hunk 1 above)
Comment (sf-jeff-lim) [reply README.md:71]: Done!

File: README.md (hunk 2 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

**Review by chrisallo — APPROVED** [review]

**Pull request merged**